### PR TITLE
FE-510

### PR DESCRIPTION
**Pull request by ipanagiv**
Removd fix location warnings

### CODE REVIEW

File: docs/project/tokenomics.mdx [hunk 1]
```diff
@@ -39,22 +39,24 @@ The following sections describe how the smart contract issues new tokens, and ho
 
 | Distribution               | Percentage  | Amount         |
 | ---------------------------| ----------- | -------------- |
-| **Station Rewards**      | 52%         | 52,000,000 WXM |
+| **Daily Station Rewards**      | 52%         | 52,000,000 WXM |
```
Comment (PavlosTze) [inline docs/project/tokenomics.mdx:42]: Let's keep "Station Rewards" as mentioned in tokenomics on the website as well? By having "Daily" here someone might think that we issue 52M daily.

File: docs/project/tokenomics.mdx [hunk 2]
```diff
@@ -39,22 +39,24 @@ The following sections describe how the smart contract issues new tokens, and ho
...
 | **Company & investors**    | 30%         | 30,000,000 WXM |
-| **Ecosystem Development**  | 10%         | 10,000,000 WXM |
+| **Treasury**  | 10%         | 10,000,000 WXM |
 | **Liquidity Support**      | 5%          | 5,000,000 WXM  |
 |**"Airdrop" to beta participants**| 3%    | 3.000.000 WXM  |
 
-#### 55% - Station Rewards:
-Tokens will be used for station owners rewards. 3M of this amount will be used to reward beta users.
+#### 55% - Daily Station Rewards:
```
Comment (PavlosTze) [inline docs/project/tokenomics.mdx:48]: Same comment about "Daily" here.

File: docs/project/introduction.mdx [hunk 3]
```diff
@@ -27,7 +27,7 @@ import useBaseUrl from "@docusaurus/useBaseUrl";
 
 ## Introduction
 
-WeatherXM is built on an EVM compatible chain (we are currently developing on Polygon, but this will be finalized near token launch) and rewards weather station owners with the WXM token. WeatherXM intends to be the world's largest, most distributed, and most decentralized weather network.
+WeatherXM is built on an EVM compatible chain. We are currently developing on [Arbitrum One](https://arbitrum.io/), and rewards weather station owners with the WXM token. WeatherXM intends to be the world's largest, most distributed, and most decentralized weather network.
```
Comment (PavlosTze) [inline docs/project/introduction.mdx:30]: Typo here, `and rewards`. I suggest keeping the initial parentheses.

**Review by PavlosTze — CHANGES_REQUESTED** [review]
@ipanagiv Except my comments below, please also remove the "relocation fee notice" from:
[https://docs.weatherxm.com/wifi-m5-bundle/m5-troubleshooting#location-not-verified](https://docs.weatherxm.com/wifi-m5-bundle/m5-troubleshooting#location-not-verified)

Also there is another "Polygon" mention at the top of in the white box:
[https://docs.weatherxm.com/tokenomics](https://docs.weatherxm.com/tokenomics)

File: docs/project/tokenomics.mdx (hunk 1 above)
Comment (ipanagiv) [reply docs/project/tokenomics.mdx:42]: ok

File: docs/project/introduction.mdx (hunk 3 above)
Comment (ipanagiv) [reply docs/project/introduction.mdx:30]: ok

**Review by PavlosTze — APPROVED** [review]

**Pull request merged**